src/MockClassifier.Api/Controllers/InstitutionController.cs

```diff
@@ -28,20 +28,25 @@ public InstitutionController(IDmrService dmrService, ITokenService tokenService,
         [HttpPost]
         public AcceptedResult Post([FromBody] MessagesInput messages)
         {
-            foreach( var message in messages.Messages)
+            if (messages == null)
+            {
+                throw new ArgumentNullException(nameof(messages));
```

Comment:
Would we want to add more relevant HTTP error handling so the calling client receives errors in HTTP context e.g. 
```suggestion
        var resp = new HttpResponseMessage(HttpStatusCode.BadRequest)
        {
            ReasonPhrase = "Message input empty"
        };
        throw new HttpResponseException(resp);
```